#Deprecated. Prefer to use the one in corobot_msgs package, which is exactly the same

uint8 connected
uint8 info
